### PR TITLE
Add custom item models tutorial

### PR DESCRIPTION
**Pull request by GitWither**

### CODE REVIEW

File: tutorials/custom-item-models.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+## The Item
+
+First of all, we need to make sure our item is ready. You can add custom models to both custom and vanilla items. In this tutorial, we will be making an automatic drill. We won't be going in depth on how to create custom items in this tutorial, but you can use [this](https://gitwither.github.io/bedrock-item-generator/) online custom item generator. Make sure your item has got a texture and displays properly in the player's inventory.
+
+![](/assets/images/tutorials/custom-item-models/drill-sprite-inventory.png)
+
+## Geometry
+
+Before implementing your custom item model into the game, make sure your item geometry is ready to be used. For your item to render correctly inside a player's hand, it must contain the player's bone skeleton. This means that your item model must contain all of the bones the player model has **without their respective elements (the player's bones must be empty!)**. Export it with a unique identifier. In our case it will be `geometry.drill`.
+
```
Comment (SirLich) [inline tutorials/custom-item-models.md:22]: Can we have an example of this json?

File: tutorials/custom-item-models.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+				{
+					"*": "Material.default"
+				}
+			]
+		}
+	}
+}
+```
+
+Pack download link: [Link](https://www.dropbox.com/s/qo4dmkmawvrvjr5/CustomItemModels.mcaddon?dl=1)
```
Comment (SirLich) [inline tutorials/custom-item-models.md:416]: You can host the file on the wiki if you want as well?

File: tutorials/custom-item-models.md [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+]
+```
+Note: we have the decorative animation defined here as well.
+
+## Final Touches
+Great! Our item works but there is a small problem: you can see that our item sprite is rendered with the 3D model in the player's hand.
+
+![](/assets/images/tutorials/custom-item-models/drill-item-visible.png)
+
+To fix this, simply set the opacity of the item texture to 1 in a free image editor like GIMP.
```
Comment (SirLich) [inline tutorials/custom-item-models.md:241]: Won't this make the "drill" in the inventory blank?

**Review by SirLich — APPROVED** [review]

File: tutorials/custom-item-models.md (hunk 3 above)
Comment (GitWither) [reply tutorials/custom-item-models.md:241]: No because Minecraft is a special kind of beast

**Pull request merged**